node/node.go

```diff
@@ -802,6 +794,21 @@ func (n *Node) apis() []rpc.API {
 			Public:    true,
 		},
 	}
+	debugRpcApi := []rpc.API{
+		{
+			Namespace: "debug",
+			Version:   "1.0",
+			Service:   debug.Handler,
+		}, {
+			Namespace: "debug",
+			Version:   "1.0",
+			Service:   NewPublicDebugAPI(n),
```

Comment:
```suggestion
			Service:   NewPublicDebugAPI(n),
			Public: true,
```

Reply:
Refer to https://github.com/klaytn/klaytn/blob/8664bc88319567f560bc5244131672d554e55293/node/node.go#L786-L790

Reply:
The terms are misleading, but..

- A `Public` service would be enabled if there is no namespace specified in the `--rpcapi` flag.
- The `PublicDebugAPI` refers to "not unsafe" debug APIs that no vulnerabilities are found.. **yet**.

So I removed the `Public` property because I thought it would be better to disable debug APIs if not explicitly specified.